sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -1,7 +1,29 @@
 # Change Log azure-storage-blob
 
-## Version 12.0.0-preview.2:
+## Version 12.0.0-preview.3 (2019-09-10):
```

Comment:
We need to add a comment about adding ability to set tier on different APIs, or am I missing it?